.flake8

```diff
@@ -39,7 +39,7 @@ per-file-ignores =
   #     F811 redefinition of unused '...'
   stdlib/typing.pyi: B, E701, E741, F401, F403, F405, F811, F822
   # Generated protobuf files include docstrings
-  *_pb2.pyi: B, E701, E741, F401, F403, F405, F822, Y021, Y026
+  *_pb2.pyi: B, E701, E741, F401, F403, F405, F822, Y021, Y026, Y053, Y054
```

Comment:
Y053 caused four flake8-pyi errors to be emitted on this monstrous line in one of the generated stub files for `tensorflow`:

https://github.com/python/typeshed/blob/8080e491d2fa92b1b1390c87fac23a1a38dcd919/stubs/tensorflow/tensorflow/core/protobuf/rewriter_config_pb2.pyi#L484